Actions/DetermineProjectsToBuild/DetermineProjectsToBuild.psm1

```diff
@@ -298,7 +298,7 @@ function Get-BuildAllProjects {
     $settings = $env:Settings | ConvertFrom-Json
 
     if (!$modifiedFiles) {
-        Write-Host "No files modified, building everything"
+        Write-Host "::notice::No files modified, building everything"
```

Comment:
Why, if there are no modified files, everything is being built?

I recall with the previous design, no modified files meant CICD is running (not PR Build).

With the new design, wouldn't it be a problem? E.g. there's a CICD that runs on a schedule with incremental builds: if there have been no changes since yesterday, suddenly everything will be built.

Reply:
Even if you haven't modified files, bc version might be changed or your dependencies. This is by design and should remain so imo.

Reply:
So, if CICD runs on a schedule:
If one day you change a single app, then only that app is built.
But if the next day you change no apps, then all of them are built?

That cannot be the design...

Reply:
I didn't change the behavior in this PR - I only change the notifications to write why it selects to do a full build due to a partner not being able to understand why it did a full build.

If CI/CD runs on a schedule - it will (by default) build all apps every time (incremental builds by default is off on a schedule). I get your point if you have selected to add incremental builds on a schedule - will make a fix for this.